lib/pbkit/.clang-format

```diff
@@ -0,0 +1,19 @@
+BasedOnStyle: Microsoft
```

Comment:
Should we have a pbkit specific format spec or eventually migrate to use "common" nxdk style? My preference is the latter

Reply:
Very strongly agreed. 

In this case my selfish goal was to avoid having to manually reformat a bunch of my code that I planned to upstream after this, so I opted to minimize the restyle to simplify the PR.